src/fosslight_dependency/package_manager/Npm.py

```
@@ -146,18 +146,17 @@ def parse_transitive_relationship(self):
         err_msg = ''
 
         cmd = 'npm ls -a --omit=dev --json -s'
-        result = subprocess.run(cmd, shell=True, capture_output=True)
-        rel_tree = decode_subprocess_output(result.stdout)
-        stderr_str = decode_subprocess_output(result.stderr)
+        result = subprocess.run(cmd, shell=True, capture_output=True, text=True, encoding='utf-8')
```

Ruff (0.15.9)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)